test/e2e/framework/pv_util.go

```diff
@@ -81,6 +81,7 @@ type PersistentVolumeConfig struct {
 	Labels           labels.Set
 	StorageClassName string
 	NodeAffinity     *v1.VolumeNodeAffinity
+	VolumeMode       v1.PersistentVolumeMode
```

Comment:
I think you should make this a ptr, and check for nil later on.  Not all tests are going to be setting volume mode.